### ISSUE TITLE
Using YOLO-LITE

### ISSUE THREAD

**Issue by Flock1** [reu2018DL/YOLO-LITE#3]
Hi,

Congrats on getting YOLO to work at 10 FPS on a CPU. I want to know how to actually use? Which model and weights are you using to get it to work at 10 FPS?

Please let me know.

**Comment by Jped** [comment]
We are using the YOLO-lit model,
the weights that achieved this result can be found here: https://github.com/reu2018DL/YOLO-LITE/blob/master/weights/tiny-yolov2-trial3-noBatch.weights
CFG can be found here : https://github.com/reu2018DL/YOLO-LITE/blob/master/cfg/tiny-yolov2-trial3-noBatch.cfg

To actually implement this check out the darknet library. https://github.com/pjreddie/darknet

**Comment by Flock1** [comment]
Thanks a lot! I want to know if the model predicts trucks or buses? I know that both COCO and PASCAL VOC have those classes but somehow, YOLO-LITE doesn't predict it. Is it a glitch or is that how the model is trained?

Also, in the paper, you wrote that on a CPU, it runs at almost 20 FPS. However, on my machine (Intel® Core™ i5-7500 CPU @ 3.40GHz × 4), the image is taking about 0.17 seconds (about 6 FPS). Can you suggest what's going on?

**Comment by rachuang22** [comment]
Hi, our model is not good at detecting certain objects (such as trucks) which is probably why it is not showing up for you. What are you using to run the real time detection to get 6 FPS?

**Comment by Flock1** [comment]
Hi. I am using basic object detection in an image. This command from the darknet website:
`./darknet detect cfg/tiny-yolov2-trial3-noBatch.cfg tiny-yolov2-trial3-noBatch.weights data/dog.jpg`
The popular dog image of YOLO. I'm running this command on my machine which has the CPU. Not doing any real-time application.

**Comment by rachuang22** [comment]
Hi, our 20 FPS is the speed of our real-time application. The speed you are referring to is just the time it takes to process one image.

**Comment by Flock1** [comment]
Hi,

I tried the implementation in real-time, using a webcam and it's working at max 6 FPS.

**Comment by Jped** [comment]
Hey again, so it really depends on what the internal specs of your computer is that you are running this on.
We tested this out on a Dell XPS 13, what are you running this on? What is your processor model? How much ram do you have?

**Comment by barzan-hayati** [comment]
> Hi, our 20 FPS is the speed of our real-time application. The speed you are referring to is just the time it takes to process one image.

@rachuang22 How running on one image take 0.17 second and in real time application reach to 0.05 second? Maybe by running on each image we need to load weight every time and it increase time of running and in real time application we don't have such situation.

**Comment by Hackinet** [comment]
Just wanted to drop this here:
Generally, running on real-time input would provide a faster FPS compared to reading from disk.

**Comment by kvin97** [comment]
Hey can we use darkflow codebase with the relevant cfg file to train the network from the begininning. Since yolo-lite similar to yolo-v2-tiny in its operations, we can do the training in darkflow also right? And I believe final layer loss function works similar to the yolo-v2-tiny from darkflow codebase.
